CMakeLists.txt

```diff
@@ -491,6 +489,8 @@ if(S2_ENABLE_INSTALL)
 endif() # S2_ENABLE_INSTALL
 
 if (BUILD_TESTS)
+  add_library(test_main "src/s2/test_main.cc")
```

Comment:
This file hasn't been added, and should probably only be added when the benchmarks are.